docker-test-framework/etc/irods-ext/metalnx.properties

```diff
@@ -175,4 +175,9 @@ pluggablepublishing.info.timeout=0
 pluggablepublishing.publishing.timeout=0
 
 # server rule engine instance that will provide the galleryview listing
-gallery_view.rule_engine_plugin.instance_name=irods_rule_engine_plugin-irods_rule_language-instance
\ No newline at end of file
+gallery_view.rule_engine_plugin.instance_name=irods_rule_engine_plugin-irods_rule_language-instance
+
+##########################################################
+# Hide metadata if attribute names start with at least one prefix.
+metalnx.metadata.mask.prefix=irods::
```

Comment:
oh, as per later comment... this should probably be plural `prefixes`

Reply:
done